src/features/Navigation/Navigation.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+import { StyledNavBar, StyledNavText, StyledNavLink, StyledLogoText, StyledNavBarItems } from './Navigation.styles';
+import Logo from './logo.svg';
+
+const AddNavBarLink = (to: string, text: string) => {
+  return (
+    <StyledNavLink to={to} className="navbar-link">
+      <div>{text}</div>
+    </StyledNavLink>
+  );
+};
+
+const NavigationBar = () => {
+  return (
+    <StyledNavBar>
+      <StyledLogoText>YLITSE</StyledLogoText>
+      <StyledNavBarItems>
+        {AddNavBarLink('/', 'Koti')}
+        {AddNavBarLink('/', 'Mentorit')}
+        {AddNavBarLink('/', 'Chat')}
+        {AddNavBarLink('/', 'Info')}
+        {AddNavBarLink('/', 'Kirjaudu ulos')}
```

Comment:
Using this renderFunction is a bit of a react antipattern

You could instead of this function just use 

```react
<StyledNavLink to={'/'}>Kirjaudu ulos</StyledNavLink>
```